.github/workflows/functional-baremetal.yaml

```diff
@@ -40,6 +40,11 @@ jobs:
     steps:
       - name: Checkout Gophercloud
         uses: actions/checkout@v4
+      - name: Workaround for grub-efi-amd64-signed
```

Comment:
I'm wondering if we could add a comment here to explain a bit why we're doing this so next time it breaks we know why we are apt-get uprading here.

This also makes me wonder if we should just enforce an apt-get upgrade in here: https://github.com/EmilienM/devstack-action/blob/main/action.yaml so we always get the latest packages.

And then I'll tag a new release and we can simply update the action tag in all workflows here.

Reply:
So, my idea to execute this was after reading 
https://forum.hestiacp.com/t/ubuntu-20-04-installation-error/4323/5 and https://askubuntu.com/questions/1276111/error-upgrading-grub-efi-amd64-signed-special-device-old-ssd-does-not-exist

I do think it makes sense to run an upgrade before trying to install devstack (normally I do this when trying to deploy).

Yup, we can just add to https://github.com/EmilienM/devstack-action/blob/main/action.yaml =)

Reply:
Sounds good; so you can submit a PR there, I'll merge and right away tag a new version. Then you can update this PR with updating all tags for the action (on all workflows using it) and we'll see how that works!

Reply:
ack, doing now